server/src/routes/api/v1/reviewsRouter.js

```diff
@@ -0,0 +1,17 @@
+import express from 'express'
+import objection from 'objection'
+const { ValidationError } = objection
+import Review from '../../../models/Review.js'
+
+const reviewsRouter = new express.Router()
+
+reviewsRouter.get('/', async (req, res) => {
```

Comment:
A page of ALL reviews is probably not needed - we typically would see the reviews for a pasta on the pasta show page, rather than a complete index of all reviews made on all pastas in the site